db/migrate/20230925100826_create_user.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] **Good job so far 👏 👏** However, The provided migration file creates a `users` table with columns for "name," "photo," and "bio," is missing the "posts_counter" column. To complete the table as intended, add `t.integer :posts_counter` within the create_table block 🙏 

![image](https://github.com/RileyManda/blog-app/assets/102757126/a3b1e2c6-ad17-4c43-8680-3fcac42f349e)